index.html

```diff
@@ -1661,6 +1671,17 @@ <h2 id="AudioContext">
           use cases, only a single <a><code>AudioContext</code></a> is used per
           document.
         </p>
+        <p>
+          An <a><code>AudioContext</code></a> is said to be <dfn>allowed to
+          start</dfn> if the user agent and the system allow sound output in
+          the current context.
+        </p>
+        <p class='note'>
+          For example, a user agent could require that an
+          <a><code>AudioContext</code></a> start is <a href=
```

Comment:
What does "start" mean here?  The meaning is really overloaded here; clarify what start really means.